### ISSUE TITLE
Safari: Non-ASCII Characters in Filename don't work when Viewing/Download PDF

### ISSUE THREAD

**Issue by jakejackson1** [blueliquiddesigns/gravity-forms-pdf-extended#1302]
**Description**
If a PDF filename contains non ASCII characters (like another language `報名成功!`) when you view or download the PDF from the admin area the filename will be garbled. 

[Research shows](https://stackoverflow.com/a/216777) there isn't a good way to resolve this using the `Content-disposition` header. The most reliable method is to include the filename at the end of the URL and not use a `Content-disposition` header at all.

Further testing will need to be done on the best way to structure the URL, and ensure it works with our existing system.

**Comment by jestonihpi** [comment]
Hi @jakejackson1. I wasn't able to replicate this issue, it works fine on my end using Chrome and Firefox browsers :
![image](https://user-images.githubusercontent.com/434866/152731465-1879243c-d19e-4de7-92d8-9595529bbd46.png)
![image](https://user-images.githubusercontent.com/434866/152731655-4da8edb6-bcae-4511-8a77-a6dda92cc7f2.png)

**Comment by jakejackson1** [comment]
@jestonihpi what's the filename look like on your local machine when you use the "download" link on the Entry Details page?

**Comment by jestonihpi** [comment]
Everything seems normal on my end @jakejackson1 :
![image](https://user-images.githubusercontent.com/434866/152885849-4fc2c416-a0f1-4aef-bc92-a6b68f8467cb.png)
![image](https://user-images.githubusercontent.com/434866/152886208-4d40ce4c-8977-4522-bebf-d234df43ee54.png)

**Comment by jakejackson1** [comment]
Further testing has shown this appears to be a Safari browser-related issue only. As you can't test this yourself to replicate, I'll look into this when I've time.

**Comment by jakejackson1** [comment]
I narrowed this problem down to a bug in Mac's Preview app, which is used in Safari to view inline PDFs in the browser. It only occurs if you view the PDF inline and then use the Preview app download button. If you use the standard PDF download link the UTF-8 filename is saved correctly. 

I tested different variations of the `Content-Disposition` header with UTF-8 support, and even removed the `filename` header option and added the name to the end of the URL. None of these standard methods of setting the filename worked as expected when the file was downloaded using the method described above. 

This is a problem that would need to be addressed by Apple.
